vaxrank/cli.py

```diff
@@ -126,6 +127,11 @@ def add_output_args(arg_parser):
         default="",
         help="Name of final reviewer of report")
 
+    output_args_group.add_argument(
+        "--output-min-epitope-report",
+        default="",
+        help="Path to the minimal epitope report")
```

Comment:
Maybe expand the help a little more. e.g. "Path to the minimal report, lists which mutant and wildtype peptides can be used for T-cell response assays"